packages/react-components/src/echarts/extensions/minMaxDataStreamValues/updateDataStreamMinMax.ts

```diff
@@ -0,0 +1,37 @@
+import { EChartsExtensionInstallRegisters } from 'echarts/types/src/extension';
+import { findMinMax } from './minMaxDataStreamSync';
+import useDataStore from '../../../store';
+
+// Echarts core use type does not map correctly to the echarts extension type so exporting as any
+// eslint-disable-next-line
+export const dataStreamMinMaxSyncExtension: any = (registers: EChartsExtensionInstallRegisters) => {
+  registers.registerUpdateLifecycle(
+    'series:afterupdate',
+    (_ecModel, _api, params) => {
+      if (!params.updatedSeries) return;
+
+      params.updatedSeries.forEach((series) => {
+        const appKitChartId = series.ecModel.option.appKitChartId as string; // widget-id
```

Comment:
You can access this once outside the loop on the _ecModel variable in the callback function.